### DISCUSSION TITLE
Translation Discussions

### DISCUSSION THREAD

**Discussion by Digitalone1** [wwmm/easyeffects#1432] [General]
@wwmm Wasn't the translation supposed to autoupdate? I completed it a couple of days ago, but the merge has not been made.

**Comment by wwmm** [comment]
It should be automatic. I will take a look at the weblate configuration page.

**Reply by wwmm** [reply]
Things seem fine in the "repository maintenance" section. I do not know what could be happening. I will wait one more day to see if something happens.

**Reply by wwmm** [reply]
Nothing happened. I will do a manual update. Maybe after this Weblate comes back to do it automatically

**Reply by wwmm** [reply]
Maybe @vchernin has some idea about what happened. I still do not see any reason why it did not do the pull request automatically even after 2 days. Nothing has changed in its configuration as far as I can see.

**Comment by wwmm** [comment]
Somehow today Weblate made the PR automatically. IF that was because I did the last one manually I have no idea.
